ddhcpd/files/etc/init.d/ddhcpd

```diff
@@ -43,7 +69,15 @@ start_service() {
   procd_append_param command -i "$server_interface"
   procd_append_param command -N "$block_network"
   procd_append_param command -H "/etc/ddhcpd.hook"
-  config_foreach dhcp_option dhcp_option 
+
+  [ "$use_config_dhcp" == "1" ] && {
+      procd_append_param command -o "1:4:$(uci get network.lan.netmask)"
+      procd_append_param command -o "3:4:$(uci get network.lan.ipaddr)"
+      local lease_time="$(uci get dhcp.lan.leasetime)"
+      local pure_lease_time="$(convert_lease_time $lease_time)"
+      procd_append_param command -o "51:4:$pure_lease_time"
+  } || config_foreach dhcp_option dhcp_option
```

Comment:
Always use loop over given options.

Reply:
Done, I'm a bit unsure how to implement the `start` and `limit` params

	option start '100'
	option limit '150'

Should this just be ignored or do you have an idea?

Reply:
@sargon I think this doesn't work as intended. Having multiple networks to server DHCP would require multiple instances of DDHCPD, should I change the script accordingly?